src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/__init__.py

```diff
@@ -30,9 +37,32 @@ def transform_extension_list_available(results):
             return [OrderedDict([('Name', r)]) for r in results]
 
         def validate_extension_add(namespace):
+            if IS_WINDOWS == False:
+                try:
+                    logger.debug('Reading from: %s', LIST_FILE_PATH)
+                    list_file = open(LIST_FILE_PATH, 'r')
+                    package_source = list_file.read() 
+                    list_file.close()
+
+                    stored_linux_dist_name = package_source.split(" ")[3]
+                    logger.debug('Found in list file: %s', stored_linux_dist_name)
+
+                    current_linux_dist_name = platform.linux_distribution()[2]
+                    logger.debug('Reported by API: %s', current_linux_dist_name)
+                except Exception as err:
+                    current_linux_dist_name = ""
+                    stored_linux_dist_name = ""
+                    logger.error('An error occurred while checking linux distribution version source list consistency.')
+                    logger.error(err)                    
```

Comment:
Prefer: `logger.debug`.
The user can't really do much if an error happens here so would rather not show anything.

Reply:
done